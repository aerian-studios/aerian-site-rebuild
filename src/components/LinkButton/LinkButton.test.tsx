/// <reference types="@types/jest" />
import * as React from "react";

import * as renderer from "react-test-renderer";

import { LinkButton } from "./index";

describe("LinkButton", () =>
    it("renders correctly", () => {
        const tree = renderer
            .create(<LinkButton to="/" className="myClass" />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    }));
