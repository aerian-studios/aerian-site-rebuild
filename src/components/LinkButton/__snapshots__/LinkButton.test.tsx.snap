// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LinkButton renders correctly 1`] = `
<a
  aria-current="page"
  className="myClass"
  href="/"
  onClick={[Function]}
  onMouseEnter={[Function]}
  style={Object {}}
/>
`;
